src/chainparams.cpp

```diff
@@ -80,6 +80,7 @@ class CMainParams : public CChainParams {
         consensus.BIP65Height = 388381; // 000000000000000004c2b624ed5d7756c508d90fd0da2c7c679febfa6c4735f0
         consensus.BIP66Height = 363725; // 00000000000000000379eaa19dce8c9b722d46ae6a57c2f1a988119488b50931
         consensus.BIP102HeightDelta = 144 * 90;
+        consensus.BIP102ReplaySunsetTime = 1518134459; // February 9, 2018
```

Comment:
Why does this behaviour expire?

Reply:
This was answered by Jeff on the main thread.  It is his opinion that by the time this date arrives one of the two forks will be the sole survivor (even though precedent of other controversial hard forks does not bear this assumption out).  He also stated that if it needs to be revised, it can be later.  However this would require yet another hard fork of the network and, as we all know, hard forks are difficult to pull off.  Personally, I agree that something so 'temporary' doesn't make sense, especially if it is dependent on yet another hard fork in the future.  In my view this should be better thought out.  Long term holders may wait many months, if not years, to split their coins.

Reply:
> However this would require yet another hard fork of the network and, as we all know, hard forks are difficult to pull off.

If we don't build in sunset, it would take a hard-fork to remove it later. Knowing how tricky hard-forks are to pull off, this would likely never happen. By building in a sunset, we can revisit later and it's only a soft-fork if we decide to renew.

The hope is that over time exchanges will develop more sophisticated solutions (tracking and comparing unspent outputs on both chains). Because `OP_RETURN` replay protections takes a little extra tx space, there will be incentive to do this. In a few months time, there will be pools of unspent outputs on both chains that could be used to taint.

Reply:
> However this would require yet another hard fork of the network and, as we all know, hard forks are difficult to pull off.

That is inaccurate. Extending the behaviour, or even making it permanent, is just a soft-fork. This means it can be enforced by a simple majority (but ideally a super-majority) of hashpower, without relying on everyone to upgrade.

Reply:
Not be confused with extending #131 protection on the 2x chain, which works the opposite way and would require a hard fork to extend IIUC.

Reply:
> Long term holders may wait many months, if not years, to split their coins.

This is precisely my concern! It places un-due urgency on making a transaction within a certain timeframe as opposed to later when it's less safe. 

Just leave it indefinitely, since the extra data isn't even hitting this chain.